### DISCUSSION TITLE
Should Ray show the dumped variable type like dd() does?

### DISCUSSION THREAD

**Discussion by ryanmortier** [spatie/ray#858] [General]
I noticed that if I dump a variable via `ray($var)`, the output is the same regardless if it's a string or integer and it bothers me that I can't used ray to determine this.

For example, the following two vars will show identical in the Ray app:

```php
$a = 1;
$b = "1";

ray($a, $b);
```

<img width="383" alt="CleanShot 2023-11-02 at 12 13 49@2x" src="https://github.com/spatie/ray/assets/2053960/01472605-8afe-4340-8847-b620f3018a37">

---

`dd()` on the other hand will show quotes around a string and uses colour coding to indicate a string or other type.

![CleanShot 2023-11-02 at 12 13 38](https://github.com/spatie/ray/assets/2053960/b0eaab69-7e37-4e0f-bcad-84a933972904)

Is there anyway to get this behaviour out of Ray?

**Comment by freekmurze** [comment]
That's a good point, we'll improve on this soon!

**Reply by ryanmortier** [reply]
Thank you! 😁
